cmd/utils/cmd.go

```diff
@@ -287,6 +291,92 @@ func ExportAppendChain(blockchain *core.BlockChain, fn string, first uint64, las
 	return nil
 }
 
+func ExportHistory(bc *core.BlockChain, dir string, first, last, step uint64) error {
```

Comment:
It's good to have a separate function operating on the blockchain object directly. 
We might be enable to provide a RPC function to dump the chain history without
shutdown.

Reply:
Sorry looks like I missed this! Are you referring to creating a function like `ExportN`? If so, two thoughts:

* this is a very special use format for pre-merge data, it isn't something that really needs to be computed by many people so I don't think it is useful to increase the surface of the blockchain api with additional methods
* the era1 format in general should probably be exposed as little as possible due to extremely limited usefulness